pages/linux/procs.md

```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
+> More information: <https://github.com/dalance/procs>.
+
+- List all processes with PID, user, CPU usage, memory usage and the command which started them:
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:
```

Comment:
What is CPU time, if that is time since it started the process then change it to 

```suggestion
- Show information about all processes sorted by running time in an [a]scending or [d]escending order:
```

Reply:
Also the orders of a and d should be switched.

Reply:
CPU time is correct. I don't know what it is exactly, but it's definitively a widely used term.
I agree with swapping ascending and descending, but nothing else:
```suggestion
- Show information about all processes sorted by CPU time in [a]scending or [d]escending order:
```